fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -310,6 +310,9 @@ struct TenantManagementWorkload : TestWorkload {
 			wait(tr->commit());
 		} else {
 			ASSERT(tenantsToCreate.size() == 1);
+			if (deterministicRandom()->coinflip()) {
+				tenantsToCreate.begin()->second.assignedCluster = self->dataClusterName;
+			}
```

Comment:
I think this does an ok job of exercising the code, but the `MetaclusterManagamentWorkload` will work better for evaluating whether it is doing the correct thing. I'd say let's leave this here and add a similar config in the other workload too.